### PR TITLE
fix: unit for vertical speed

### LINKED ISSUE #5395: VSI indicators looks locked at 0 ftm
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-07-24T20:18:52+00:00",
  "ref": "master",
  "sha": "05a0bbda844850568c82ae6dab6217e509a46aa2",
  "actor": "davidwalschots",
  "event_name": "manual"
}
```

### Describe the bug

The VSI on the PFD of both the captain and first officer seems stuck at 0 ftm (they actually move, but very little). It seems they've been applied the wrong multiplier. On external view, the virtual VSI  works as expected.

### Expected behavior

The VSI should display the correct vertical speed.

### Steps to reproduce

1. Spawn anywhere
2. Pitch up or down
3. Watch as VSI indicator moves very little as attitude changes.

### References (optional)

_No response_

### Additional info (optional)

I've missed a few PRs, so I can't tell on which one exactly this problem started for me.

### Discord Username (optional)

CassiKess

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #5395

## Summary of Changes

The sim provides the variable `VELOCITY WORLD Y` in` feet per second`, whereas `feet per minute` was assumed. This causes e.g. the PFD VS indication to show wrong values. 
I replaced `foot per minute` with `foot per second` to mitigate this. 
Feel free to close this PR if the solution is not satisfying.

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/systems/systems/src/navigation/adirs.rs [hunk 1]
```diff
@@ -12,7 +12,7 @@ use uom::si::{
     f64::*,
     length::foot,
     thermodynamic_temperature::degree_celsius,
-    velocity::{foot_per_minute, knot},
+    velocity::{foot_per_minute, foot_per_second, knot},
```
Comment (ZigTag) [inline src/systems/systems/src/navigation/adirs.rs:15]: Unused import

**Review by ZigTag — CHANGES_REQUESTED** [review]
Just need to remove the unused import

File: src/systems/systems/src/navigation/adirs.rs (hunk 1 above)
Comment (davidwalschots) [reply src/systems/systems/src/navigation/adirs.rs:15]: There are other usages.

**Comment by davidwalschots** [comment]
The PDF originally also used feet per minute:

image

**Comment by davidwalschots** [comment]
Found the culprit. This isn't it. 
In `a320_systems_wasm/src/lib.rs` #5242 added (merged yesterday 10:49Z):
```rust
reader.add("VELOCITY WORLD Y", "feet per minute", 0)?;
```
and #5159 added below it (merged yesterday 20:08Z):
```rust
reader.add("VELOCITY WORLD Y", "Feet per Second", 0)?;
```

Thus `feet per second` is read, whereas #5242 assumes `feet per minute`.

**Comment by davidwalschots** [comment]
I'll work on a fix.

**Comment by davidwalschots** [comment]
See #5398.

**Pull request closed without merging**